app/controllers/subscriber/group_controller.rb

```diff
@@ -82,5 +82,17 @@ def replace_validation_errors
       end
     end
 
+    def subscription_tags
+      [collect_tags, collect_tags(excluded: true)].flatten.map do |tag|
```

Comment:
`[collect_tags, collect_tags(excluded: true)]` -> create method or variable all_tags / tags ? 

tags = collect_tags + collect_excluded_tags

makes it more readable